security/aws/rds_publicly_accessible/README.md

```diff
@@ -1,75 +1,81 @@
- 
-When a publicly accessible RDS instance is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have an option to modify configuration after manual approval, and even Users can perform delete action if required. 
-- *remove public access rule* - modifies the configuration of the RDS instance by disabling the public access rule
- 
-### Input Parameters
- 
-- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
-- *Ignore tags* - RDS instances with any of these tags will be ignored 
- 
-### Required RightScale Roles
- 
-- policy_manager
-- admin or credential_viewer
-
-### AWS Required Permissions
 
-This policy requires permissions to describe AWS RDS instances, list RDS tags, modify RDS instances and delete RDS instances.
-The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. The IAM user credentials contained in those credentials will require the following permissions:
+## Functional Details
 
-```javascript
-{
-  "Version": "2014-10-31",
-  "Statement":[{
-  "Effect":"Allow",
-  "Action":["rds:DescribeDBInstances",
-            "rds:ListTagsForResource",
-            "rds:ModifyDBInstance",
-            "rds:CreateDBClusterSnapshot",
-            "rds:DescribeDBClusterSnapshots",			
-            "rds:DeleteDBInstance"],
-    "Resource":"*"
-    }
-  ]
-}
-```
+When a publicly accessible RDS instance is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have an option to modify configuration after manual approval, and even Users can perform delete action if required.
 
-### Supported Clouds
- 
-- AWS
- 
-### Cost
- 
-This Policy Template does not incur any cloud costs.
+- *remove public access rule* - modifies the configuration of the RDS instance by disabling the public access rule
+
+## Input Parameters
+
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
+- *Tags to ignore* - List of tags that will exclude resources from being evaluated by this policy. Multiple tags are evaluated as an 'OR' condition. Tag keys or key/value pairs can be listed. Example: 'test,env=dev'
 
+## Policy Actions
 
-### Enable delete action
+- Sends an email notification
+- Disable the publicly accessible RDS instances after approval
+- Delete publicly accessible RDS instances after approval (optional)
 
 Perform below steps to enable delete action.
 
-- Edit the file [AWS_Publicly_Accessible_RDS_Instances](https://github.com/rightscale/policy_templates/tree/master/security/aws/rds_publicly_accessible/AWS_Publicly_Accessible_RDS_Instances.pt)
+- Edit the file [AWS_Publicly_Accessible_RDS_Instances](https://github.com/flexera/policy_templates/tree/master/security/aws/rds_publicly_accessible/aws_publicly_accessible_rds_instances.pt)
```

Comment:
to fix the 404 error in the danger file for this url remove this part of the link **https://github.com/flexera/policy_templates/tree/master**

Reply:
Removed mentioned part from the URL